### PR TITLE
feat(): add new CFT fields for starter templates

### PR DESCRIPTION
**Pull request by Ceseura**

### CODE REVIEW

File: default/resources/km/field/team.json [hunk 1]
```diff
@@ -1,21 +1,8 @@
...
-            "supportedEntityTypeIds": [
-              "financialProfessional"
-            ],
-            "type": "ONE_WAY"
-          }
-        }
-      }
-    },
-    "typeId": "list"
+    "typeId": "c_teamSection"
```
Comment (bhainesva) [inline default/resources/km/field/team.json:7]: Is the teamSection cft that we're referencing here defined somewhere? Same for the other fields.

Comment (Ceseura) [reply default/resources/km/field/team.json:7]: Yes I created them all here & updated the entities to use those fields https://sandbox.yext.com/s/3160439/account/customFieldTypes

Comment (bhainesva) [reply default/resources/km/field/team.json:7]: sorry so the changes in this pr to `default/resources/km/field/` create new Custom Fields. The `typeId` references a field type or custom field type. 

When we reference a custom field type, that type itself needs to be defined, like it's done for `c_hero` here: https://github.com/yextconsulting/solution-pages-starter-consulting/blob/main/default/resources/km/field-type/hero.json#L2

This repo doesn't have a direct connection to the sandbox testing account so the things being defined there isn't directly relevant

Comment (bhainesva) [reply default/resources/km/field/team.json:7]: But if they are defined in the sbx account it should hopefully be easy to copy paste the field type definitions here

**Review by bhainesva — APPROVED** [review]

**Pull request merged**